flytekit/loggers.py

```diff
@@ -1,29 +1,60 @@
-import logging as _logging
-import os as _os
+import logging
+import os
 
 from pythonjsonlogger import jsonlogger
 
-logger = _logging.getLogger("flytekit")
-# Always set the root logger to debug until we can add more user based controls
-logger.setLevel(_logging.WARNING)
+# Note:
+# The environment variable controls exposed to affect the individual loggers should be considered to be beta.
+# The ux/api may change in the future.
+# At time of writing, the code was written to preserve existing default behavior
+# For now, assume this is the environment variable whose usage will remain unchanged and controls output for all
+# loggers defined in this file.
+LOGGING_ENV_VAR = "FLYTE_SDK_LOGGING_LEVEL"
+
+# By default, the root flytekit logger to debug so everything is logged, but enable fine-tuning
+logger = logging.getLogger("flytekit")
+# Root logger control
+flytekit_root_env_var = f"{LOGGING_ENV_VAR}_ROOT"
```

Comment:
why introduce this env var? What's the use case for having `FLYTE_SDK_LOGGING_LEVEL` different than `FLYTE_SDK_LOGGING_LEVEL_ROOT`?

Reply:
yeah the former controls the handler (which serves all loggers), the latter controls the root flytekit logger itself.